qt/widgets/common/inc/MantidQtWidgets/Common/Spectra.h

```diff
@@ -0,0 +1,104 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include "DllOption.h"
+#include "IndexTypes.h"
+#include "MantidKernel/ArrayProperty.h"
+
+#include <set>
+#include <stdexcept>
+#include <vector>
+
+#include <QList>
+#include <QString>
+#include <QStringList>
+
+namespace MantidQt {
+namespace MantidWidgets {
+
+/*
+ * Representation of a discontinuous spectra range.
+ * Can be used in a vector-like manner.
+ *
+ * Holds a string and vector representation.
+ */
+class EXPORT_OPT_MANTIDQT_COMMON Spectra {
```

Comment:
Sorry that I didn't spot this the first time but as we have renamed DataSet, and this has Fit-related things in it, can we revert this name back to `FunctionModelSpectra`?